lib/kuzzle/kuzzle.ts

```diff
@@ -299,6 +339,145 @@ class Kuzzle extends KuzzleEventEmitter {
     }
   }
 
+  // For testing purpose
+  async ask (...args: any[]) {
+    return super.ask(...args);
+  }
+
+  // For testing purpose
+  async emit (...args: any[]) {
+    return super.emit(...args);
+  }
+
+  // For testing purpose
+  async pipe (...args: any[]) {
+    return super.pipe(...args);
+  }
+
+  private async _importUserMappings(config: {
```

Comment:
The first `_` is useless with typescript (we used it to indicate a private method)
Apply to other private methods